src/Config/ConfigLoader.php

```diff
@@ -32,12 +33,13 @@ function load(string $workingDir): Config
 
         $data = \array_replace_recursive(self::DEFAULTS, $data);
 
-        $data['image'] !== null
-            or U::fail('Image name is not specified');
+        $data['image'] !== null or $data['container'] !== null
+            or U::fail('Image or container name is not specified');
 
         return new Config(
             $baseDir,
-            $data['image'],
+            $data['image'] ?: '',
```

Comment:
```suggestion
            $data['image'],
```

Shouldn't be needed when `Config` constructor accepts `null`s. Same with `$data['container']`.